src/containers/CartItem/CartItem.js

```diff
@@ -51,7 +52,6 @@ function CartItem(props) {
   const { product } = cartItem
 
   const { decreaseItem, increaseItem, removeItem } = useCentraHandlers()
-  const { t } = useI18n()
```

Comment:
You have to use useLingui hook if you are using `t` macro inside a react components. Otherwise component will not be updated when language is switched.  Or use <Trans> react macro, which will automatically subscribed for changes.  

So for smoother migration i will suggest to replace all `const { t } = useI18n()` to `useLingui()` (even if the result of it is not used)